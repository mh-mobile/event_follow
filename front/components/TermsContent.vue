<template lang="pug">
  div
    h2 利用規約
    p 本利用規約（以下「本規約」と言います。）には、このウェブサイト上で提供するサービス(以下、「本サービスといいます。」)の提供条件及び当サイトと登録ユーザーの皆様との間の権利義務関係が定められています。本サービスの利用に際しては、本規約の全文をお読みいただいたうえで、本規約に同意いただく必要があります。
    h3 第1条（適用）
    ol
      li 本規約は、本サービスの提供条件及び本サービスの利用に関する当サイトと登録ユーザーとの間の権利義務関係を定めることを目的とし、登録ユーザーと当サイトとの間の本サービスの利用に関わる一切の関係に適用されます。
      li 当サイト上で掲載する本サービス利用に関するルールは、本規約の一部を構成するものとします。
      li 本規約は、本サービスの提供条件及び本サービスの利用に関する当サイトと登録ユーザーとの間の権利義務関係を定めることを目的とし、登録ユーザーと当サイトとの間の本サービスの利用に関わる一切の関係に適用されます。
    h3 第2条（定義）
    p 本規約において使用する以下の用語は、各々以下に定める意味を有するものとします。
    ol.bracket-number
      li 「サービス利用契約」とは、本規約を契約条件として当サイトと登録ユーザーの間で締結される、本サービスの利用契約を意味します。
      li 「知的財産権」とは、著作権、特許権、実用新案権、意匠権、商標権その他の知的財産権（それらの権利を取得し、またはそれらの権利につき登録等を出願する権利を含みます。）を意味します。
      li 「登録ユーザー」とは、第3条（登録）に基づいて本サービスの利用者としての登録がなされた個人を意味します。
      li 「当サイト」とは、そのドメインが「event-follow-front.herokuapp.com」であるウェブサイト（理由の如何を問わず、当ウェブサイトのドメインまたは内容が変更された場合は、当該変更後のウェブサイトを含みます。）を意味します。
      li 「本サービス」とは、当サイトが提供する「EVENT FOLLOW」という名称のサービス（理由の如何を問わずサービスの名称または内容が変更された場合は、当該変更後のサービスを含みます。）を意味します。
    h3 第3条（登録）
    ol
      li 本サービスの利用を希望する者（以下「登録希望者」といいます。）は、本規約を遵守することに同意し、かつ当サイトの定める一定の情報（以下「登録事項」といいます。）を当サイトの定める方法で当サイトに提供することにより、当サイトに対し、本サービスの利用の登録を申請することができます。
      li 当サイトは、当サイトの基準に従って、第１項に基づいて登録申請を行った登録希望者（以下「登録申請者」といいます。）の登録の可否を判断し、当サイトが登録を認める場合にはその旨を登録申請者に通知します。登録申請者の登録ユーザーとしての登録は、当サイトが本項の通知を行ったことをもって完了したものとします。
      li 前項に定める登録の完了時に、サービス利用契約が登録ユーザーと当サイトの間に成立し、登録ユーザーは本サービスを本規約に従い利用することができるようになります。
      li 当サイトは、登録申請者が、以下の各号のいずれかの事由に該当する場合は、登録及び再登録を拒否することがあり、またその理由について一切開示義務を負いません。
      ol.bracket-number
        li 当サイトに提供した登録事項の全部または一部につき虚偽、誤記または記載漏れがあった場合
        li 反社会的勢力等（暴力団、暴力団員、右翼団体、反社会的勢力、その他これに準ずる者を意味します。以下同じ。）である、または資金提供その他を通じて反社会的勢力等の維持、運営もしくは経営に協力もしくは関与する等反社会的勢力等との何らかの交流もしくは関与を行っていると当サイトが判断した場合
        li 過去当サイトとの契約に違反した者またはその関係者であると当サイトが判断した場合
        li 第8条に定める措置を受けたことがある場合
        li その他、登録を適当でないと当サイトが判断した場合
    h3 第4条（パスワード及びユーザーIDの管理）
    ol
      li 登録ユーザーは、自己の責任において、本サービスに関するパスワード及びユーザーIDを適切に管理及び保管するものとし、これを第三者に利用させ、または貸与、譲渡、名義変更、売買等をしてはならないものとします。
      li パスワードまたはユーザーIDの管理不十分、使用上の過誤、第三者の使用等によって生じた損害に関する責任は登録ユーザーが負うものとします。
    h3 第5条（禁止事項）
    p 登録ユーザーは、本サービスの利用にあたり、以下の各号のいずれかに該当する行為または該当すると当サイトが判断する行為をしてはなりません。
    ol 
      li 法令に違反する行為または犯罪行為に関連する行為
      li 当サイト、本サービスの他の利用者またはその他の第三者に対する詐欺または脅迫行為
      li 公序良俗に反する行為
      li 当サイト、本サービスの他の利用者またはその他の第三者の知的財産権、肖像権、プライバシーの権利、名誉、その他の権利または利益を侵害する行為
      li 本サービスを通じ、以下に該当し、または該当すると当サイトが判断する情報を当サイトまたは本サービスの他の利用者に送信すること
      ul
        li 過度に暴力的または残虐な表現を含む情報
        li コンピューター・ウィルスその他の有害なコンピューター・プログラムを含む情報
        li 当サイト、本サービスの他の利用者またはその他の第三者の名誉または信用を毀損する表現を含む情報
        li 過度にわいせつな表現を含む情報
        li 差別を助長する表現を含む情報
        li 自殺、自傷行為を助長する表現を含む情報
        li 薬物の不適切な利用を助長する表現を含む情報
        li 反社会的な表現を含む情報
        li 他人に不快感を与える表現を含む情報
        li 本サービスのネットワークまたはシステム等に過度な負荷をかける行為
        li 当サイトが提供するソフトウェアその他のシステムに対するリバースエンジニアリングその他の解析行為
        li 本サービスの運営を妨害するおそれのある行為
        li 当サイトのネットワークまたはシステム等への不正アクセス
        li 第三者に成りすます行為
        li 本サービスの他の利用者のIDまたはパスワードを利用する行為
        li 当サイトが事前に許諾しない本サービス上での宣伝、広告、勧誘、または営業行為
        li 本サービスの他の利用者の情報の収集
        li 当サイト、本サービスの他の利用者またはその他の第三者に不利益、損害、不快感を与える行為
        li 反社会的勢力等への利益供与
        li 面識のない異性との出会いを目的とした行為
        li 前各号の行為を直接または間接に惹起し、または容易にする行為
        li 前各号の行為を試みること
        li その他、当サイトが不適切と判断する行為

    h3 第6条（本サービスの停止等）
    p 当サイトは、以下のいずれかに該当する場合には、登録ユーザーに事前に通知することなく、本サービスの全部または一部の提供を停止または中断することができるものとします。
    ol.bracket-number
      li 本サービスに係るコンピューター・システムの点検または保守作業を緊急に行う場合
      li コンピューター、通信回線等の障害、誤操作、過度なアクセスの集中、不正アクセス、ハッキング等により本サービスの運営ができなくなった場合
      li 地震、落雷、火災、風水害、停電、天災地変などの不可抗力により本サービスの運営ができなくなった場合
      li その他、当サイトが停止または中断を必要と判断した場合
    
    h3 第7条（権利帰属）
    ol
      li 当サイト及び本サービスに関する知的財産権は全て当サイトまたは当サイトにライセンスを許諾している者に帰属しており、本規約に基づく本サービスの利用許諾は、本サービスに関する当サイトまたは当サイトにライセンスを許諾している者の知的財産権の使用許諾を意味するものではありません。

    h3 第8条（登録抹消等）
    ol
      li 当サイトは、登録ユーザーが、以下の各号のいずれかの事由に該当する場合は、事前に通知または催告することなく、当該登録ユーザーについて本サービスの利用を一時的に停止し、または登録ユーザーとしての登録を抹消することができます。
      ol.bracket-number
        li 本規約のいずれかの条項に違反した場合
        li 登録事項に虚偽の事実があることが判明した場合
        li 第3条第4項各号に該当する場合
        li その他、当サイトが本サービスの利用または登録ユーザーとしての登録の継続を適当でないと判断した場合
      li 前項各号のいずれかの事由に該当した場合、登録ユーザーは、当サイトに対して負っている債務の一切について当然に期限の利益を失い、直ちに当サイトに対して全ての債務の支払を行わなければなりません。

    h3 第9条（退会）
    ol
      li 登録ユーザーは、当サイト所定の手続の完了により、本サービスから退会し、自己の登録ユーザーとしての登録を抹消することができます。
      li 退会にあたり、当サイトに対して負っている債務が有る場合は、登録ユーザーは、当サイトに対して負っている債務の一切について当然に期限の利益を失い、直ちに当サイトに対して全ての債務の支払を行わなければなりません。
      li 退会後の利用者情報の取扱いについては、第13条の規定に従うものとします。
    
    h3 第10条（本サービスの内容の変更、終了）
    ol
      li 当サイトは、当サイトの都合により、本サービスの内容を変更し、または提供を終了することができます。
      li 当サイトが本サービスの提供を終了する場合、当サイトは登録ユーザーに事前に通知せずに終了することができるものとします。

    h3 第11条（保証の否認及び免責）
    ol 
      li 当サイトは、本サービスが登録ユーザーの特定の目的に適合すること、期待する機能・商品的価値・正確性・有用性を有すること、登録ユーザーによる本サービスの利用が登録ユーザーに適用のある法令または業界団体の内部規則等に適合すること、継続的に利用できること、及び不具合が生じないことについて、明示又は黙示を問わず何ら保証するものではありません。
      li 当サイトは、本サービスに関して登録ユーザーが被った損害につき、賠償する責任を負わないものとし、また、付随的損害、間接損害、特別損害、将来の損害及び逸失利益にかかる損害については、賠償する責任を負わないものとします。
      li 本サービスまたは当サイトに関連して登録ユーザーと他の登録ユーザーまたは第三者との間において生じた取引、連絡、紛争等については、登録ユーザーが自己の責任によって解決するものとします。
    h3 第12条（秘密保持）
    p 登録ユーザーは、本サービスに関連して当サイトが登録ユーザーに対して秘密に取扱うことを求めて開示した非公知の情報について、当サイトの事前の書面による承諾がある場合を除き、秘密に取扱うものとします。
    h3 第13条（利用者情報の取扱い）
    ol
      li 当サイトによる登録ユーザーの利用者情報の取扱いについては、別途当サイトプライバシーポリシーの定めによるものとし、登録ユーザーはこのプライバシーポリシーに従って当サイトが登録ユーザーの利用者情報を取扱うことについて同意するものとします。
      li 当サイトは、登録ユーザーが当サイトに提供した情報、データ等を、個人を特定できない形での統計的な情報として、当サイトの裁量で、利用及び公開することができるものとし、登録ユーザーはこれに異議を唱えないものとします。
    h3 第14条（本規約等の変更）
    p 当サイトは、当サイトが必要と認めた場合は、本規約を変更できるものとします。本規約を変更する場合、変更後の本規約の施行時期及び内容を当サイトウェブサイト上での掲示その他の適切な方法により周知し、または登録ユーザーに通知します。但し、法令上登録ユーザーの同意が必要となるような内容の変更の場合は、当サイト所定の方法で登録ユーザーの同意を得るものとします。

    h3 第15条（連絡／通知）
    ol
      li 本サービスに関する問い合わせその他登録ユーザーから当サイトに対する連絡または通知、及び本規約の変更に関する通知その他当サイトから登録ユーザーに対する連絡または通知は、当サイトの定める方法で行うものとします。
      li 当サイトが登録事項に含まれるメールアドレスその他の連絡先に連絡または通知を行った場合、登録ユーザーは当該連絡または通知を受領したものとみなします。
    h3 第16条（サービス利用契約上の地位の譲渡等）
    ol
      li 登録ユーザーは、当サイトの書面による事前の承諾なく、利用契約上の地位または本規約に基づく権利もしくは義務につき、第三者に対し、譲渡、移転、担保設定、その他の処分をすることはできません。
      li 当サイトは本サービスにかかる事業を他社に譲渡した場合には、当該事業譲渡に伴い利用契約上の地位、本規約に基づく権利及び義務並びに登録ユーザーの登録事項その他の顧客情報を当該事業譲渡の譲受人に譲渡することができるものとし、登録ユーザーは、かかる譲渡につき本項において予め同意したものとします。なお、本項に定める事業譲渡には、通常の事業譲渡のみならず、会社分割その他事業が移転するあらゆる場合を含むものとします。

    h3 第17条（分離可能性）
    p 本規約のいずれかの条項またはその一部が、消費者契約法その他の法令等により無効または執行不能と判断された場合であっても、本規約の残りの規定及び一部が無効または執行不能と判断された規定の残りの部分は、継続して完全に効力を有するものとします。

    h3 第18条（準拠法及び管轄裁判所）
    ol
      li 本規約及びサービス利用契約の準拠法は日本法とします。
      li 本規約またはサービス利用契約に起因し、または関連する一切の紛争については、東京地方裁判所を第一審の専属的合意管轄裁判所とします。

    p 2021年1月29日制定


</template>

<style lang="scss" scoped>
h1 {
  font-size: 2em;
  margin: 0.67em 0;
  font-weight: bold;
}

h2 {
  font-size: 1.5em;
  margin: 0.75em 0;
  font-weight: bold;
}

h3 {
  font-size: 1.17em;
  margin: 0.83em 0;
  font-weight: bold;
}

ol {
  padding: 0;
  margin: 20px;
  li {
    padding-left: 1em;
    line-height: 1.5;
    padding-top: 5px;
  }
}

ol.bracket-number {
  padding: 0;
  margin: 20px;

  li {
    padding-left: 1em;
    line-height: 1.5;
    padding-top: 5px;
    list-style-type: none;
    counter-increment: cnt;
  }

  li::before {
    display: marker;
    content: "(" counter(cnt) ") ";
  }
}

ul {
  padding: 0;
  margin: 20px;
  li {
    list-style-type: disc;
    padding-left: 1em;
    line-height: 1.5;
    padding-top: 5px;
  }
}
</style>
