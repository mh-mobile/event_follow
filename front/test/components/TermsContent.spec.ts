import { createLocalVue, shallowMount } from "@vue/test-utils"
import VueCompositionApi from "@vue/composition-api"
import TermsContent from "@/components/TermsContent.vue"

const localVue = createLocalVue();
localVue.use(VueCompositionApi);
describe("TermsContent.vue", () => {
  it("renders html correctly", () => {
    const wrapper = shallowMount(TermsContent, {
      localVue,
    });
    expect(wrapper.html()).toMatchSnapshot();
  });
});
