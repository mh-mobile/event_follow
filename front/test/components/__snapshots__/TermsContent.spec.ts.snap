// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TermsContent.vue renders html correctly 1`] = `
"<div>
  <h2>利用規約</h2>
  <p>本利用規約（以下「本規約」と言います。）には、このウェブサイト上で提供するサービス(以下、「本サービスといいます。」)の提供条件及び当サイトと登録ユーザーの皆様との間の権利義務関係が定められています。本サービスの利用に際しては、本規約の全文をお読みいただいたうえで、本規約に同意いただく必要があります。</p>
  <h3>第1条（適用）</h3>
  <ol>
    <li>本規約は、本サービスの提供条件及び本サービスの利用に関する当サイトと登録ユーザーとの間の権利義務関係を定めることを目的とし、登録ユーザーと当サイトとの間の本サービスの利用に関わる一切の関係に適用されます。</li>
    <li>当サイト上で掲載する本サービス利用に関するルールは、本規約の一部を構成するものとします。</li>
    <li>本規約は、本サービスの提供条件及び本サービスの利用に関する当サイトと登録ユーザーとの間の権利義務関係を定めることを目的とし、登録ユーザーと当サイトとの間の本サービスの利用に関わる一切の関係に適用されます。</li>
  </ol>
  <h3>第2条（定義）</h3>
  <p>本規約において使用する以下の用語は、各々以下に定める意味を有するものとします。</p>
  <ol class=\\"bracket-number\\">
    <li>「サービス利用契約」とは、本規約を契約条件として当サイトと登録ユーザーの間で締結される、本サービスの利用契約を意味します。</li>
    <li>「知的財産権」とは、著作権、特許権、実用新案権、意匠権、商標権その他の知的財産権（それらの権利を取得し、またはそれらの権利につき登録等を出願する権利を含みます。）を意味します。</li>
    <li>「登録ユーザー」とは、第3条（登録）に基づいて本サービスの利用者としての登録がなされた個人を意味します。</li>
    <li>「当サイト」とは、そのドメインが「event-follow-front.herokuapp.com」であるウェブサイト（理由の如何を問わず、当ウェブサイトのドメインまたは内容が変更された場合は、当該変更後のウェブサイトを含みます。）を意味します。</li>
    <li>「本サービス」とは、当サイトが提供する「EVENT FOLLOW」という名称のサービス（理由の如何を問わずサービスの名称または内容が変更された場合は、当該変更後のサービスを含みます。）を意味します。</li>
  </ol>
  <h3>第3条（登録）</h3>
  <ol>
    <li>本サービスの利用を希望する者（以下「登録希望者」といいます。）は、本規約を遵守することに同意し、かつ当サイトの定める一定の情報（以下「登録事項」といいます。）を当サイトの定める方法で当サイトに提供することにより、当サイトに対し、本サービスの利用の登録を申請することができます。</li>
    <li>当サイトは、当サイトの基準に従って、第１項に基づいて登録申請を行った登録希望者（以下「登録申請者」といいます。）の登録の可否を判断し、当サイトが登録を認める場合にはその旨を登録申請者に通知します。登録申請者の登録ユーザーとしての登録は、当サイトが本項の通知を行ったことをもって完了したものとします。</li>
    <li>前項に定める登録の完了時に、サービス利用契約が登録ユーザーと当サイトの間に成立し、登録ユーザーは本サービスを本規約に従い利用することができるようになります。</li>
    <li>当サイトは、登録申請者が、以下の各号のいずれかの事由に該当する場合は、登録及び再登録を拒否することがあり、またその理由について一切開示義務を負いません。</li>
    <ol class=\\"bracket-number\\">
      <li>当サイトに提供した登録事項の全部または一部につき虚偽、誤記または記載漏れがあった場合</li>
      <li>反社会的勢力等（暴力団、暴力団員、右翼団体、反社会的勢力、その他これに準ずる者を意味します。以下同じ。）である、または資金提供その他を通じて反社会的勢力等の維持、運営もしくは経営に協力もしくは関与する等反社会的勢力等との何らかの交流もしくは関与を行っていると当サイトが判断した場合</li>
      <li>過去当サイトとの契約に違反した者またはその関係者であると当サイトが判断した場合</li>
      <li>第8条に定める措置を受けたことがある場合</li>
      <li>その他、登録を適当でないと当サイトが判断した場合</li>
    </ol>
  </ol>
  <h3>第4条（パスワード及びユーザーIDの管理）</h3>
  <ol>
    <li>登録ユーザーは、自己の責任において、本サービスに関するパスワード及びユーザーIDを適切に管理及び保管するものとし、これを第三者に利用させ、または貸与、譲渡、名義変更、売買等をしてはならないものとします。</li>
    <li>パスワードまたはユーザーIDの管理不十分、使用上の過誤、第三者の使用等によって生じた損害に関する責任は登録ユーザーが負うものとします。</li>
  </ol>
  <h3>第5条（禁止事項）</h3>
  <p>登録ユーザーは、本サービスの利用にあたり、以下の各号のいずれかに該当する行為または該当すると当サイトが判断する行為をしてはなりません。</p>
  <ol>
    <li>法令に違反する行為または犯罪行為に関連する行為</li>
    <li>当サイト、本サービスの他の利用者またはその他の第三者に対する詐欺または脅迫行為</li>
    <li>公序良俗に反する行為</li>
    <li>当サイト、本サービスの他の利用者またはその他の第三者の知的財産権、肖像権、プライバシーの権利、名誉、その他の権利または利益を侵害する行為</li>
    <li>本サービスを通じ、以下に該当し、または該当すると当サイトが判断する情報を当サイトまたは本サービスの他の利用者に送信すること</li>
    <ul>
      <li>過度に暴力的または残虐な表現を含む情報</li>
      <li>コンピューター・ウィルスその他の有害なコンピューター・プログラムを含む情報</li>
      <li>当サイト、本サービスの他の利用者またはその他の第三者の名誉または信用を毀損する表現を含む情報</li>
      <li>過度にわいせつな表現を含む情報</li>
      <li>差別を助長する表現を含む情報</li>
      <li>自殺、自傷行為を助長する表現を含む情報</li>
      <li>薬物の不適切な利用を助長する表現を含む情報</li>
      <li>反社会的な表現を含む情報</li>
      <li>他人に不快感を与える表現を含む情報</li>
      <li>本サービスのネットワークまたはシステム等に過度な負荷をかける行為</li>
      <li>当サイトが提供するソフトウェアその他のシステムに対するリバースエンジニアリングその他の解析行為</li>
      <li>本サービスの運営を妨害するおそれのある行為</li>
      <li>当サイトのネットワークまたはシステム等への不正アクセス</li>
      <li>第三者に成りすます行為</li>
      <li>本サービスの他の利用者のIDまたはパスワードを利用する行為</li>
      <li>当サイトが事前に許諾しない本サービス上での宣伝、広告、勧誘、または営業行為</li>
      <li>本サービスの他の利用者の情報の収集</li>
      <li>当サイト、本サービスの他の利用者またはその他の第三者に不利益、損害、不快感を与える行為</li>
      <li>反社会的勢力等への利益供与</li>
      <li>面識のない異性との出会いを目的とした行為</li>
      <li>前各号の行為を直接または間接に惹起し、または容易にする行為</li>
      <li>前各号の行為を試みること</li>
      <li>その他、当サイトが不適切と判断する行為</li>
    </ul>
  </ol>
  <h3>第6条（本サービスの停止等）</h3>
  <p>当サイトは、以下のいずれかに該当する場合には、登録ユーザーに事前に通知することなく、本サービスの全部または一部の提供を停止または中断することができるものとします。</p>
  <ol class=\\"bracket-number\\">
    <li>本サービスに係るコンピューター・システムの点検または保守作業を緊急に行う場合</li>
    <li>コンピューター、通信回線等の障害、誤操作、過度なアクセスの集中、不正アクセス、ハッキング等により本サービスの運営ができなくなった場合</li>
    <li>地震、落雷、火災、風水害、停電、天災地変などの不可抗力により本サービスの運営ができなくなった場合</li>
    <li>その他、当サイトが停止または中断を必要と判断した場合</li>
  </ol>
  <h3>第7条（権利帰属）</h3>
  <ol>
    <li>当サイト及び本サービスに関する知的財産権は全て当サイトまたは当サイトにライセンスを許諾している者に帰属しており、本規約に基づく本サービスの利用許諾は、本サービスに関する当サイトまたは当サイトにライセンスを許諾している者の知的財産権の使用許諾を意味するものではありません。</li>
  </ol>
  <h3>第8条（登録抹消等）</h3>
  <ol>
    <li>当サイトは、登録ユーザーが、以下の各号のいずれかの事由に該当する場合は、事前に通知または催告することなく、当該登録ユーザーについて本サービスの利用を一時的に停止し、または登録ユーザーとしての登録を抹消することができます。</li>
    <ol class=\\"bracket-number\\">
      <li>本規約のいずれかの条項に違反した場合</li>
      <li>登録事項に虚偽の事実があることが判明した場合</li>
      <li>第3条第4項各号に該当する場合</li>
      <li>その他、当サイトが本サービスの利用または登録ユーザーとしての登録の継続を適当でないと判断した場合</li>
    </ol>
    <li>前項各号のいずれかの事由に該当した場合、登録ユーザーは、当サイトに対して負っている債務の一切について当然に期限の利益を失い、直ちに当サイトに対して全ての債務の支払を行わなければなりません。</li>
  </ol>
  <h3>第9条（退会）</h3>
  <ol>
    <li>登録ユーザーは、当サイト所定の手続の完了により、本サービスから退会し、自己の登録ユーザーとしての登録を抹消することができます。</li>
    <li>退会にあたり、当サイトに対して負っている債務が有る場合は、登録ユーザーは、当サイトに対して負っている債務の一切について当然に期限の利益を失い、直ちに当サイトに対して全ての債務の支払を行わなければなりません。</li>
    <li>退会後の利用者情報の取扱いについては、第13条の規定に従うものとします。</li>
  </ol>
  <h3>第10条（本サービスの内容の変更、終了）</h3>
  <ol>
    <li>当サイトは、当サイトの都合により、本サービスの内容を変更し、または提供を終了することができます。</li>
    <li>当サイトが本サービスの提供を終了する場合、当サイトは登録ユーザーに事前に通知せずに終了することができるものとします。</li>
  </ol>
  <h3>第11条（保証の否認及び免責）</h3>
  <ol>
    <li>当サイトは、本サービスが登録ユーザーの特定の目的に適合すること、期待する機能・商品的価値・正確性・有用性を有すること、登録ユーザーによる本サービスの利用が登録ユーザーに適用のある法令または業界団体の内部規則等に適合すること、継続的に利用できること、及び不具合が生じないことについて、明示又は黙示を問わず何ら保証するものではありません。</li>
    <li>当サイトは、本サービスに関して登録ユーザーが被った損害につき、賠償する責任を負わないものとし、また、付随的損害、間接損害、特別損害、将来の損害及び逸失利益にかかる損害については、賠償する責任を負わないものとします。</li>
    <li>本サービスまたは当サイトに関連して登録ユーザーと他の登録ユーザーまたは第三者との間において生じた取引、連絡、紛争等については、登録ユーザーが自己の責任によって解決するものとします。</li>
  </ol>
  <h3>第12条（秘密保持）</h3>
  <p>登録ユーザーは、本サービスに関連して当サイトが登録ユーザーに対して秘密に取扱うことを求めて開示した非公知の情報について、当サイトの事前の書面による承諾がある場合を除き、秘密に取扱うものとします。</p>
  <h3>第13条（利用者情報の取扱い）</h3>
  <ol>
    <li>当サイトによる登録ユーザーの利用者情報の取扱いについては、別途当サイトプライバシーポリシーの定めによるものとし、登録ユーザーはこのプライバシーポリシーに従って当サイトが登録ユーザーの利用者情報を取扱うことについて同意するものとします。</li>
    <li>当サイトは、登録ユーザーが当サイトに提供した情報、データ等を、個人を特定できない形での統計的な情報として、当サイトの裁量で、利用及び公開することができるものとし、登録ユーザーはこれに異議を唱えないものとします。</li>
  </ol>
  <h3>第14条（本規約等の変更）</h3>
  <p>当サイトは、当サイトが必要と認めた場合は、本規約を変更できるものとします。本規約を変更する場合、変更後の本規約の施行時期及び内容を当サイトウェブサイト上での掲示その他の適切な方法により周知し、または登録ユーザーに通知します。但し、法令上登録ユーザーの同意が必要となるような内容の変更の場合は、当サイト所定の方法で登録ユーザーの同意を得るものとします。</p>
  <h3>第15条（連絡／通知）</h3>
  <ol>
    <li>本サービスに関する問い合わせその他登録ユーザーから当サイトに対する連絡または通知、及び本規約の変更に関する通知その他当サイトから登録ユーザーに対する連絡または通知は、当サイトの定める方法で行うものとします。</li>
    <li>当サイトが登録事項に含まれるメールアドレスその他の連絡先に連絡または通知を行った場合、登録ユーザーは当該連絡または通知を受領したものとみなします。</li>
  </ol>
  <h3>第16条（サービス利用契約上の地位の譲渡等）</h3>
  <ol>
    <li>登録ユーザーは、当サイトの書面による事前の承諾なく、利用契約上の地位または本規約に基づく権利もしくは義務につき、第三者に対し、譲渡、移転、担保設定、その他の処分をすることはできません。</li>
    <li>当サイトは本サービスにかかる事業を他社に譲渡した場合には、当該事業譲渡に伴い利用契約上の地位、本規約に基づく権利及び義務並びに登録ユーザーの登録事項その他の顧客情報を当該事業譲渡の譲受人に譲渡することができるものとし、登録ユーザーは、かかる譲渡につき本項において予め同意したものとします。なお、本項に定める事業譲渡には、通常の事業譲渡のみならず、会社分割その他事業が移転するあらゆる場合を含むものとします。</li>
  </ol>
  <h3>第17条（分離可能性）</h3>
  <p>本規約のいずれかの条項またはその一部が、消費者契約法その他の法令等により無効または執行不能と判断された場合であっても、本規約の残りの規定及び一部が無効または執行不能と判断された規定の残りの部分は、継続して完全に効力を有するものとします。</p>
  <h3>第18条（準拠法及び管轄裁判所）</h3>
  <ol>
    <li>本規約及びサービス利用契約の準拠法は日本法とします。</li>
    <li>本規約またはサービス利用契約に起因し、または関連する一切の紛争については、東京地方裁判所を第一審の専属的合意管轄裁判所とします。</li>
  </ol>
  <p>【2021年1月29日制定】</p>
</div>"
`;
